lib/ASRouter.jsm

```diff
@@ -705,9 +705,11 @@ class _ASRouter {
     return state;
   }
 
-  async _addPreviewEndpoint(url) {
+  async _addPreviewEndpoint(url, target) {
```

Comment:
just a nit: change the parameter name to `portID` since that's what you're passing in